ui/ozone/platform/wayland/wayland_pointer.cc

```diff
@@ -176,4 +176,8 @@ int WaylandPointer::GetFlagsWithKeyboardModifiers() {
   return flags_;
 }
 
+void WaylandPointer::ResetFlags() {
+  flags_ = 0;
```

Comment:
please reset keyboard modifiers as well, because it may result in an undesirable behavior.

Reply:
done